src/pages/Possibility/KivaContentBlock.vue

```diff
@@ -142,39 +144,85 @@ $cta-color: #611b1f;
 
 		.action-button {
 			background-color: $cta-color;
-			box-shadow: darken($cta-color, 10%) !important;
+			box-shadow: 0 2px darken($cta-color, 10%);
 
 			&:hover,
 			&:focus {
-				background-color: darken($cta-color, 10%) !important;
+				background-color: darken($cta-color, 10%);
 			}
 		}
 	}
-}
 
-.slide-placeholder {
-	// background-color: red;
-	width: 100%;
-	padding-bottom: 600/480 * 100%;
+	// This color customization isn't working for the second slide, because the 
+	// kv-carousel uses the css display: none; property which removes the element from
+	// the DOM which then messes up the ability to apply css using :nth-child(#)
+
```

Comment:
I'm open to suggestions here if anyone has them.

Reply:
Alternatively we could just keep all the text and buttons the same color and remove all this extra css. Brandon left a comment in the ticket that reads, "Brandon Smith to update colors to be consistent" So maybe the color changes between slides was something that was supposed to be removed from the ticket?

Reply:
@BoulderBrains One option would be to add a unique class to each kv-carousel-slide as you loop through your data. Then you could use that class instead of nth-childing to customize the colors.

Reply:
☝️

Reply:
👍